CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to the Kowainik repositories
+
+## :wave: Greetings Traveler!
+
+We are delighted you're reading this, and we appreciate the effort you're
+taking to make our projects awesome! :sparkles:
+
+## How to contribute
+
+### :bug: Report bugs or feature request :bulb:
+
+If you discover a bug or have any proposals on how to make this project better
+don't hesitate to create an issue [here](../../issues/new) in a free format.
+
+### Create a PR
+
+We love to receive pull requests from everyone! It's usually a good idea
+to tell about your intention to work on something under the corresponding
+issue, so everyone is aware that you're on it. If there's no such issue — simply
+create a new one!
+
+To get started with the Pull Request implementation you should first
+[fork](../../fork), then clone the repo:
+
+    git clone git@github.com:your-username/project-name.git
+
+Make your changes and consider the following checklist to go through
+before submitting your pull request.
+
+### :white_check_mark: Check list
+
+- [ ] New/fixed features work as expected (Bonus points for the new tests).
+- [ ] There are no warnings during compilation.
+- [ ] `hlint .` output is: _No Hints_ (see [`hlint`][hlint] tool docs).
```

Comment:
Hehe, in some future we will change this line to `PR approved by hintman` :smirk: